src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -15,7 +15,7 @@
     public static final Prefix PREFIX_ROOMNUMBER = new Prefix("rn/");
     public static final Prefix PREFIX_TIER = new Prefix("ti/");
     public static final Prefix PREFIX_FROMDATE = new Prefix("df/");
-    public static final Prefix PREFIX_TODATE = new Prefix("dt/");
+    public static final Prefix PREFIX_TODATE = new Prefix("td/");
```

Comment:
not sure about this change

Reply:
the prefix is called `TO_DATE` so i figured `td` is more intuitive compared to `dt` (there was inconsistency between our UG and app too, which is why many people reported they couldn't check in).

i realised i should change `df` to `fd` too if that's the case - will do so next!